README.md

```diff
@@ -22,6 +22,8 @@ This app is created by the volunteers of [Public Knowledge Workshop](https://www
 - clone the repo
 - `yarn install`
 - `yarn start`
+- If you get an error with `yarn start` then open terminal, go to the root of the project and run this command - `Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope Process`. For more info, go to this link - https://bobbyhadz.com/blog/yarn-cannot-be-loaded-running-scripts-disabled
```

Comment:
I I understand correctly, it's not a particular setting for this project. Setting this policy is needed for every Yarn project execution on any Windows environment.
If that's the case, I don't think it worth mentioning in our `README` file

Reply:
Alright. I will remove this line.